wherehows-web/app/components/visualization/charts/score-gauge.ts

```diff
@@ -56,7 +56,7 @@ export default class VisualizationChartsScoreGauge extends Component {
    * @type {number}
    * @default 0
    */
-  score: number;
+  score: number = this.score || 0;
```

Comment:
Once this issue is fixed, there's a potential this will break
https://github.com/Microsoft/TypeScript/issues/5987
, if you need a default value it's best to do it in the constructor
```ts
  
  score: number;

  constructor() {
    super(...arguments);
    
    // typeof this.score === 'number' || (this.score = 0);
    // OR
    typeof this.score === 'number' || set(this, 'score', 0);
  }
```

Reply:
Looks like this is left over code since I've just looked at the `constructor`